applegacy/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">appLegacy</string>
```

Comment:
Use the app name here? Or update the manifest to not use this and use the app name string?

Reply:
These values files will be removed in the favour of the actual relevant value files, I'll leave a todo to remove them, for now it is updated to be Forzenbook